test/websocket.ganache.js

```diff
@@ -64,9 +62,8 @@ describe('WebsocketProvider (ganache)', function () {
         await web3.eth.getBlockNumber();
 
         await new Promise(async function(resolve){
-            web3.currentProvider.once('error', function(err){
-                assert(err.message.includes('1012'));
-                assert(err.message.includes('restart'));
+            web3.currentProvider.once('close', function(event){
+                assert.strictEqual(event.code, 1000);
```

Comment:
Believe this test should continue work as expected. The disconnect uses an error code. Test was added in #3486.